students/woogyeong/westagram/users/views.py

```diff
@@ -1 +1,34 @@
-from django.shortcuts import render
+# python built-in module
+import json
+
+# 외부 module
+from django.http  import JsonResponse
+from django.views import View
+
+# 사용자 module (직접 작성한 module)
+from users.models import User
+from users.validators import validate_email, validate_password
+
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            if not validate_email(data['email']):
+                return JsonResponse({'message': 'Email format is not valid'}, status=400)
+            
+            if not validate_password(data['password']):
+                return JsonResponse({'message': 'Password format is not valid'}, status=400)
+            
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message': 'USER_EXISTS'}, status=400)
+            User.objects.create(
+                    email         = data['email'],
+                    mobile        = data['mobile'],
+                    user_name     = data['user_name'],
+                    user_id       = data['user_id'],
+                    password      = data['password'],
+                )
```

Comment:
```suggestion
            if User.objects.filter(email=data['email']).exists():
                return JsonResponse({'message': 'USER_EXISTS'}, status=400)
            
            User.objects.create(
                    email     = data['email'],
                    mobile    = data['mobile'],
                    user_name = data['user_name'],
                    user_id   = data['user_id'],
                    password  = data['password'],
            )
```
1. if가 끝난 뒤에는 한 칸 공백을 주세요.
2. =을 기준으로 정렬할 때는 가장 긴 변수명 기준으로 한 칸만 띄어 주세요.
3. )은 (가 시작하는 곳 과 맞추면 좋습니다.